pkgs/top-level/python-packages.nix

```diff
@@ -554,6 +554,8 @@ in {
 
   serversyncstorage = callPackage ../development/python-modules/serversyncstorage {};
 
+  shellingham = callPackage ../development/python-modules/shellingham {};
```

Comment:
Can you put those into individual commits?

Reply:
done, also disabled the failing python2 build